src/views/Drive/components/VersionHistory/components/CurrentVersionItem.tsx

```diff
@@ -0,0 +1,28 @@
+import dateService from 'services/date.service';
+import { Avatar } from '@internxt/ui';
+import { FileVersion } from '../types';
+
+interface CurrentVersionItemProps {
+  version: FileVersion;
+}
+
+export const CurrentVersionItem = ({ version }: CurrentVersionItemProps) => {
+  return (
+    <div className="group flex items-start justify-between border-b-[2.5px] border-gray-5 px-5 py-3 hover:bg-gray-1 dark:hover:bg-gray-5">
+      <div className="flex min-w-0 flex-1 flex-col space-y-1">
+        <div className="flex items-center justify-between">
+          <span className="text-base font-semibold text-gray-100">
+            {dateService.format(version.date, 'MMM D, h:mm A')}
```

Comment:
Extract that to a variable, something like `const versionDate = dateService.format...`